client-js/app/elements/labrad-plot1d.ts

```diff
@@ -105,13 +110,20 @@ export class Plot1D extends polymer.Base implements Plot {
             .attr('height', height + p. margin.top + p.margin.bottom)
             .append('svg:g')
             .attr('transform', `translate(${p.margin.left}, ${p.margin.top})`)
-            .call(p.zoom);
-
+            .call(p.zoom)
+	    .append('svg:g')
+	    .on('mousedown', p.zoomRectangle.bind(p));
```

Comment:
indentation looks off here and a few places below. Is there maybe a mix of tabs and spaces?